### PR TITLE
annotation with cuboids updated

### PR DESCRIPTION
**Pull request by mdacoca**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: site/content/en/docs/manual/basics/3d-object-annotation.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+| Up/down         | **Al**t+**U**/ **Alt**+**O**                  |
+| Zoom in/ou      | **Alt**+**K**/ **Alt**+**I**                  |
+
+<!--lint enable maximum-line-length-->
+
+## Annotation with cuboids
+
+There are two options available for 3D annotation:
+
+- **Shape**: for tasks like object detection, segmentation, and classification.
```
Comment (bsekachev) [inline site/content/en/docs/manual/basics/3d-object-annotation.md:48]: detection - fine
segmentation is not supported for 3D
for classification we use tags, they also not supported on 3D scene

File: site/content/en/docs/manual/basics/3d-object-annotation.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+- **Track**: uses interpolation to predict the position of objects in subsequent frames of images or videos.
```
Comment (bsekachev) [inline site/content/en/docs/manual/basics/3d-object-annotation.md:49]:
```suggestion
- **Track**: uses interpolation to predict the position of objects in subsequent frames.
```

File: site/content/en/docs/manual/basics/3d-object-annotation.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  A unique ID will be assigned to each object and maintained throughout the video or set of images.
```
Comment (bsekachev) [inline site/content/en/docs/manual/basics/3d-object-annotation.md:50]:
```suggestion
  A unique ID will be assigned to each object and maintained throughout the sequence of images.
```

File: site/content/en/docs/manual/basics/3d-object-annotation.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+To add a 3D shape, do the following:
+
+1. On the objects pane, select **Draw new cuboid** >
+   select the label from the drop-down list > **Shape**.
+
+![](/images/image217.jpg)
+
+2. The cursor will be followed by a cuboid.
+   Place the cuboid on the 3D object.
```
Comment (bsekachev) [inline site/content/en/docs/manual/basics/3d-object-annotation.md:62]:
```suggestion
   Place the cuboid on the 3D scene.
```

File: site/content/en/docs/manual/basics/3d-object-annotation.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+### Tracking with cuboids
+
+To track with cuboids, do the following:
+
+1. On the objects pane, select **Draw new cuboid** >
+   select the label from the drop-down list > **Track**.
+
+2. The cursor will be followed by a cuboid.
+   Place the cuboid on the 3D object.
```
Comment (bsekachev) [inline site/content/en/docs/manual/basics/3d-object-annotation.md:88]:
```suggestion
   Place the cuboid on the 3D scene.
```

File: site/content/en/docs/manual/basics/3d-object-annotation.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+![](/images/gif027_carla_town3.gif)
+
+4. (Optional) Move one of the four points to change the size of the cuboid.
+
+![](/images/gif028_carla_town3.gif)
+
+5. (Optional) To rotate the cuboid, click on the middle point
+   and then drag the cuboid up/down or to left/right.
+
+![](/images/gif029_carla_town3.gif)
```
Comment (bsekachev) [inline site/content/en/docs/manual/basics/3d-object-annotation.md:102]: These 4 steps are the same like for shapes, do you think we should somehow combine them or better to keep as is?

File: site/content/en/docs/manual/basics/3d-object-annotation.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+7. Move several frames forward. You will see the cuboid you've added in frame 1.
+   Adjust it, if needed.
+
+8. Repeat to the last frame with the presence of the object you are tracking.
```
Comment (bsekachev) [inline site/content/en/docs/manual/basics/3d-object-annotation.md:107]:
```suggestion
6. Move several frames forward. You will see the cuboid you've added in frame 1.
   Adjust it, if needed.

7. Repeat to the last frame with the presence of the object you are tracking.
```

File: site/content/en/docs/manual/basics/3d-object-annotation.md (hunk 6 above)
Comment (mdacoca) [reply site/content/en/docs/manual/basics/3d-object-annotation.md]: The thing is that if a person wants to annotate with trackers he will jump straight to the tracker's procedure. He will not read the part from shapes and another part from trackers, this is why there is a duplication.

File: site/content/en/docs/manual/basics/3d-object-annotation.md (hunk 1 above)
Comment (mdacoca) [reply site/content/en/docs/manual/basics/3d-object-annotation.md:48]: updated in the doc

File: site/content/en/docs/manual/advanced/contextual-images.md [hunk 8]
```diff
@@ -28,7 +28,7 @@ Before uploading the archive to CVAT, do the following:
 1. In the folder with the images for annotation, create a folder: `related_images`.
 2. Add to the `related_images` a subfolder with the same name
    as the primary image to which it should be linked.
-3. Place the context image(s) within the subfolder created in step 2.
+3. Place the contextual image(s) within the subfolder created in step 2.
```
[the remark this answers is not in the document]
Comment (mdacoca) [reply site/content/en/docs/manual/advanced/contextual-images.md:31]: updated in the doc

File: site/content/en/docs/manual/advanced/contextual-images.md [hunk 9]
```diff
@@ -62,22 +62,69 @@ Example file structure for 2D and 3D tasks:
...
+        /0000000001.bin
+        /0000000002.bin
+        /0000000003.bin
+{{< /tab >}}
 {{< /tabpane >}}
 
+- For KITTI: `image_00`, `image_01`, `image_02`, `image_0N`,
+(where `N` is any number <= 12) are context images.
+- For 3D option 2: a regular image file placed near
+a .pcd file with the same name is considered to be a context image.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/contextual-images.md:111]:
```suggestion
- For 3D option 3: a regular image file placed near
a .pcd file with the same name is considered to be a context image.
```

File: site/content/en/docs/manual/advanced/contextual-images.md [hunk 10]
```diff
@@ -62,22 +62,69 @@ Example file structure for 2D and 3D tasks:
...
+/velodyne_points
+    /data
+        /0000000000.bin
+        /0000000001.bin
+        /0000000002.bin
+        /0000000003.bin
+{{< /tab >}}
 {{< /tabpane >}}
 
+- For KITTI: `image_00`, `image_01`, `image_02`, `image_0N`,
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/contextual-images.md:108]:
```suggestion
/image_N
    /data
        /0000000000.png
        /0000000001.png
        /0000000002.png
        /0000000003.png
/velodyne_points
    /data
        /0000000000.bin
        /0000000001.bin
        /0000000002.bin
        /0000000003.bin
{{< /tab >}}
{{< /tabpane >}}

- For KITTI: `image_00`, `image_01`, `image_02`, `image_N`,
```

**Review by bsekachev — APPROVED** [review]

**Pull request merged**